webplugin/template/mck-sidebox.html

```diff
@@ -337,6 +337,11 @@ <h3>No more messages!</h3>
 					<span id="mck-other-queries">If you have any other queries, just</span> <a id="mck-restart-conversation" href="javaScript:void(0)">restart this conversation</a>
 				</div>
 				<div id = "mck-conversation-status-box" class= "n-vis mck-row"></div>
+			    <div class="mck-char-warning n-vis">
+					<div>
+						<p class="mck-char-warning-text">Keep your message within 256 characters to help the bot understand easily  |  <span class="mck-char-count"></span></p>
```

Comment:
@hardikparashar this need to come from MCK_LABELS so that localization strings can be applied https://docs.kommunicate.io/docs/web-localization

Reply:
resolved in new commit